osquery/core/database/database.cpp

```diff
@@ -0,0 +1,51 @@
+/**
+ *  Copyright (c) 2018-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/core/database/database.h>
+
+namespace osquery {
+
+Expected<int32_t, DatabaseError> Database::getInt32(const std::string& domain, const std::string& key) {
```

Comment:
This types `int32_t`, `size_t` and so far is a part of standard library. And because of that it is better to use them with namespace prefix `std::`. You still could use it without prefix because of mess up with common `C` and `C++` old libraries and backward compatibility. Just to be neat.

Reply:
std::int32_t is to verbose as for basic POD type :) 
and collision of names is nearly impossible here